lib/pronto/commentator/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Pronto
   module CommentatorVersion
-    VERSION = '0.1.0'
+    VERSION = '0.1.1'
```

Comment:
I'm not really an expert in semver, but technically wouldn't dependency changes be transitive? So a "minor" bump in the dependency would result in a "minor" bump here (i.e. to 0.2.0)?